autogpt_platform/backend/backend/util/typeshift.py

```diff
@@ -0,0 +1,147 @@
+from __future__ import annotations
```

Comment:
nit: this seems like tests would be a good idea

Reply:
I gave up on this idea, I'll just stick to DTO as return types for all RPC